### PR TITLE
Feature/inventory service

### PR DESCRIPTION
**Pull request by raaedk**
basic implementation of inventory service with mock class

### CODE REVIEW

**Comment by GannonDeWalt** [comment]
Import for vehicle model is wrong, fixing now

File: src/app/tests/inventory.service.spec.ts [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+
+  it('should be created', () => {
+    expect(service).toBeTruthy();
+  });
+
+  it('getAll() should get all vehicles in inventory list', () => {
+    spyOn(httpService, 'get').and.returnValue(of(mockVehicles));
+
+    service.getAllVehicles().subscribe((data) => {
+      console.log(data);
```
Comment (rmuehler) [inline src/app/tests/inventory.service.spec.ts:38]: ?

**Review by rmuehler — COMMENTED** [review]
Good stuff!

**Review by rmuehler — APPROVED** [review]
I like this, let's pull it!!!!!

File: src/app/models/Vehicle.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  model: string;
+  year: number;
+  miles: number;
+  color: string;
+  images: string[];
+  vin: string;
+  offerPrice: number;
+  sellingPrice: number;
+  seller: number;
+}
```
Comment (rmuehler) [inline src/app/models/Vehicle.ts:12]: Do we still need this?

Comment (rmuehler) [reply src/app/models/Vehicle.ts:12]: *this file

File: src/app/models/vehicle.ts [hunk 3]
```diff
@@ -1,3 +1,14 @@
-export class Vehicle {
-  //TODO: fill this out with vehicle info
-}
\ No newline at end of file
+export default class Vehicle {
```
Comment (wangscs) [inline src/app/models/vehicle.ts:1]: Why are there two models for vehicle?
models/vehicles.ts
models/Vehicles.ts

File: src/app/models/Vehicle.ts (hunk 2 above)
Comment (raaedk) [reply src/app/models/Vehicle.ts]: there's a bug or something, the deletion of it wont commit

**Comment by wangscs** [comment]
Looks good!

**Review by wangscs — APPROVED** [review]

**Review by ammitchky — APPROVED** [review]
service setup looks good, passes all tests

**Comment by raaedk** [comment]
dont approve this yet it's bugged into a different dimension on our local from that vehicle ts change

**Comment by rmuehler** [comment]
> dont approve this yet it's bugged into a different dimension on our local from that vehicle ts change

Didn't see this, sorry.  Pushing to the branch might reopen it again?

**Comment by raaedk** [comment]
umm i think it's fine? just remerge it and ill repull it, i think vscode is just broken and the code is fine

**Pull request merged**